forge/test/mlir/mnist/training/test_training.py

```diff
@@ -21,6 +21,92 @@
 def test_mnist_training():
     torch.manual_seed(0)
 
+    # Model and data type. To use bfloat16 follow the instructions found at test_forge_vs_torch() in this file.
```

Comment:
test_forge_vs_torch is moving into tt-thomas - it makes sense to copy the instructions here.

Reply:
Makes sense, done here de73a93e9f1d67ef233382203264d7929ea23cc4